build/scripts/entrypoint-volume.sh

```diff
@@ -25,7 +25,6 @@ ls -la /checode/
 # Start the machine-exec component in background
 export MACHINE_EXEC_PORT=3333
 nohup /checode/bin/machine-exec --url "0.0.0.0:${MACHINE_EXEC_PORT}" &
```

Comment:
@RomanNikitenko it would be nice to remove the trailing `&` which is redundant now.

Reply:
> If a command is terminated by the control operator &, the shell executes the command in the background in a subshell. The shell does not wait for the command to finish, and the return status is 0.


do we really want to wait starting `machine-exec` here?
do you see a real use case when a terminal is not ready after starting IDE?

Reply:
you're right - running it asynchronously sounds perfect
Please, ignore my note. Sorry for the noise.